.env

```diff
@@ -1 +0,0 @@
-NEXT_PUBLIC_API_BASE_URL=https://api-production.ironfish.network
```

Comment:
I removed this file from source control and added the env value to our Vercel deployment settings.